hacking/installdeps.sh

```diff
@@ -33,9 +48,11 @@ done
 
 # Install packages:
 if [ -n "${PACKAGES_REQUIRED}" ]; then
+    apt -y update
```

Comment:
Probably makes sense to do this once, regardless.  We know that one set or the other will be non-empty (if not both).